pwiz_tools/Skyline/TestPerf/PerfMobilIonTest.cs

```diff
@@ -80,6 +81,25 @@ void Test(string docFile, int groups, int peptides, int tranGroups, int transiti
             var document = WaitForDocumentLoaded();
             AssertEx.IsDocumentState(document, null, groups, peptides, tranGroups, transitions);
 
+            // Change the first precursor so that it has explicit drift time instead of explicit CCS
+            RunUI(() =>
+            {
+                // Select the first precursor
+                SkylineWindow.SequenceTree.SelectedNode = SkylineWindow.SequenceTree.Nodes[0].FirstNode.Nodes[0];
+            });
+            var editMoleculeDlgA = ShowDialog<EditCustomMoleculeDlg>(SkylineWindow.ModifySmallMoleculeTransitionGroup);
+            RunUI(() =>
+            {
+                editMoleculeDlgA.IonMobility = 308.71;
+                editMoleculeDlgA.IonMobilityUnits = eIonMobilityUnits.drift_time_msec;
+                editMoleculeDlgA.CollisionalCrossSectionSqA = null;
+            });
+            OkDialog(editMoleculeDlgA, editMoleculeDlgA.OkDialog);
+            document = WaitForDocumentChange(document);
+            var im = document.MoleculePrecursorPairs.First().NodeGroup.ExplicitValues;
+            AssertEx.AreEqual(308.71, im.IonMobility, .01); // As set above
+            AssertEx.IsFalse(im.CollisionalCrossSectionSqA.HasValue); // As set during chromatogram extraction
```

Comment:
I'm confused about what this line of code is checking.
The value that gets set during chromatogram extraction is "TransitionChromInfo.IonMobility.CollisionalCrossSectionSqA".
This assert is checking TransitionDocNode.ExplicitValues.CollisionalCrossSectionSqA which wouldn't change during chromatogram extraction.

Reply:
Misleading comment, sorry. Cut and paste cruft, I'll fix it.